### DISCUSSION TITLE
Add option to toggle the sidebar on "files changed"

### DISCUSSION THREAD

**Discussion by Davekennedy111** [community/community#61320] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

It's already hard enough to review code side-by-side in the browser. Please give me the option to not waste all that screen space for an omnipresent list of files changed...

**Comment by amarzot** [comment]
<img width="326" alt="image" src="https://github.com/community/community/assets/43010352/1d0d24a2-d402-4c47-b060-02f14700d4b5">

**Comment by Sahnesiguzel33** [comment]
nasılyani

**Comment by Spixmacaw** [comment]
Sidebar
GitLens adds many side bar views to provide additional rich functionality. There are three side bars with GitLens views in them: GitLens Inspect, GitLens, and Source Control.

All views are free to use on all repos, except for views:

marked with a ✨ require a [trial or paid plan](https://www.gitkraken.com/gitlens/pricing) for use on privately hosted repos
marked with a ☁️ require a GitKraken Account, with access level based on your [plan](https://www.gitkraken.com/gitlens/pricing), e.g. Free, Pro, etc
Side Bars
GitLens Inspect
GitLens Inspect side bar focuses on providing contextual information and insights to what you’re actively working on. This side bar includes:

Commit Details
Line History
File History
Visual File History
Search & Compare

![image](https://help.gitkraken.com/wp-content/uploads/gl-inspect-side-bar.png)
